src/utils/EmailToken.js

```diff
@@ -31,7 +31,6 @@ const ResetToken = user => {
 // Access token required for a user
 const UseraccessRequired = (req, res, next) => {
   const { token } = req.params;
-  const { email } = req.body;
   const decodedToken = jwt.decode(token, process.env.JWT_SECRET);
 
```

Comment:
Here check if the `decodedToken` is valid because it returns an `Application Error` when `decodedToken === undefined`.